Neos.Neos/Classes/Eel/FlowQueryOperations/SortOperation.php

```diff
@@ -91,10 +109,32 @@ public function evaluate(FlowQuery $flowQuery, array $arguments)
         }
 
         // Create the sort sequence
+        $sortFlags = SORT_REGULAR;
+        foreach ($sortOptions as $sortOpt) {
+            switch (strtoupper($sortOpt)) {
+                case 'SORT_FLAG_CASE':
+                    if ($sortFlags & (SORT_STRING | SORT_LOCALE_STRING | SORT_NATURAL)) {
+                        $sortFlags |= SORT_FLAG_CASE;
+                    }
+                    break;
+                case 'SORT_NUMERIC':
+                    $sortFlags = SORT_NUMERIC;
+                    break;
+                case 'SORT_STRING':
+                    $sortFlags = SORT_STRING;
+                    break;
+                case 'SORT_LOCALE_STRING':
+                    $sortFlags = SORT_LOCALE_STRING;
+                    break;
+                case 'SORT_NATURAL':
+                    $sortFlags = SORT_NATURAL;
+                    break;
+            }
+        }
```

Comment:
I would remove the additional logic and the loop and would simply: 
```suggestion
        $sortFlags = array_sum(array_map('constant', $sortOptions));
        $sortFlags = $sortFlags === 0 ? SORT_REGULAR : $sortFlags;
```

Reply:
...I'm not too firm with all these eincredible PHP aray funtions, Great solution.